using-mobile-extensions/adobe-media-analytics/README.md

```diff
@@ -178,32 +207,33 @@ public class MobileApp extends Application {
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
 In your app's `application:didFinishLaunchingWithOptions`, register Media with Mobile Core:
 
+### Swift
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+   MobileCore.registerExtensions([Media.self, Analytics.self, Identity.self], {
+   MobileCore.configureWith(appId: "yourAppId") 
+ })  
+ ...
+}
+```
 ### Objective-C
 
 ```objectivec
-#import "ACPCore.h"
-#import "ACPAnalytics.h"
-#import "ACPIdentity.h"
-#import "ACPMedia.h"
-
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    [ACPCore setLogLevel:ACPMobileLogLevelDebug];
-    [ACPCore configureWithAppId:@"your-launch-app-id"];
-
-    [ACPAnalytics registerExtension];
-    [ACPIdentity registerExtension];
-    [ACPMedia registerExtension];
-
-    [ACPCore start:^{
-
-    }];
-
-    return YES;
-  }
+    [AEPMobileCore registerExtensions:@[AEPMobileMedia.class, AEPMobileAnalytics.class, AEPMobileIdentity.class] completion:^{
+    [AEPMobileCore configureWithAppId: @"yourAppId"];
+  }];
+  ...
+}
 ```
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+In your app's `application:didFinishLaunchingWithOptions`, register Media with Mobile Core:
```

Comment:
Same edit question as above:
Should the function signature be styled as:
`application(_:didFinishLaunchingWithOptions:)`
to follow the Swift conventions for function signatures? [StackOverflow discussion](https://stackoverflow.com/questions/54537403/what-is-the-meaning-of-the-in-swift-function-signatures-and-why-is-there-no-co)